### PR TITLE
Fix to few bugs

### PR DESCRIPTION
**Pull request by Bahamas20**
Fixed bugs:
#207 
#209
#228 
#176 
#195 
#200

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,7 +66,7 @@ dependencies {
 }
 
 shadowJar {
-    archiveFileName = 'addressbook.jar'
+    archiveFileName = 'salesy.jar'
```
Comment (hauchongtang) [inline build.gradle:69]: Yes, good catch here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Adds an item of a specified type with the given details
 
 ***Format:*** 
 
-`add [n/NAME] [p/PHONE] [pr/PRICE] [i/ITEM] [a/ADDRESS] [t/Supplier]` (supplier)
+`add [n/NAME] [p/PHONE(8 digits)] [pr/PRICE] [i/ITEM] [a/ADDRESS] [t/Supplier]` (supplier)
```
Comment (hauchongtang) [inline docs/UserGuide.md:37]: Good addition for user clarity.

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -20,19 +20,19 @@ public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a supplier to the address book. "
-            + "Parameters: "
+            + "\nParameters: "
```
Comment (hauchongtang) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:23]: 👍

File: src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java [hunk 4]
```diff
@@ -36,7 +36,11 @@ public AddTaskCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddTaskCommand.MESSAGE_USAGE));
         }
 
-        String description = argMultimap.getValue(PREFIX_DESCRIPTION).get();
+        String description = argMultimap.getValue(PREFIX_DESCRIPTION).get().replaceAll("\\s+", " ");
```
Comment (hauchongtang) [inline src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java:39]: What does this regex do ?

File: src/main/java/seedu/address/model/person/Phone.java [hunk 5]
```diff
@@ -11,8 +11,8 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be 8 digits long";
+    public static final String VALIDATION_REGEX = "\\d{8}";
```
Comment (hauchongtang) [inline src/main/java/seedu/address/model/person/Phone.java:15]: Good inclusion

**Review by hauchongtang — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts too !

File: src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java (hunk 4 above)
Comment (Bahamas20) [reply src/main/java/seedu/address/logic/parser/AddTaskCommandParser.java:39]: To trim whitespaces inbetween words for task description

**Review by hauchongtang — APPROVED** [review]
LGTM

**Comment by weesiau** [comment]
Resolves #206

**Pull request merged**